File: java/begin/bmi.java
import java.util.Scanner;

public class BMI {
  public static void main(String... args) {
    Scanner sc = new Scanner(System.in);
    System.out.print("Zadej svou vysku v metrech: ");
    double vyska = sc.nextDouble();
    System.out.print("Zadej svou vahu v kilogramech: ");
    double vaha = sc.nextDouble();

    double BMI = vaha / (vyska * vyska);
    System.out.println("Tvoje BMI je " + BMI);
  }
}

File: cpp/begin/bmi.cpp
#include <iostream>
using namespace std;

int main()
{
  cout << "Zadej svou vysku v metrech: ";
  float vyska;
  cin >> vyska;

  cout << "Zadej svou vahu v kilogramech: ";
  float vaha;
  cin >> vaha;

  float BMI = vaha / (vyska * vyska);

  cout << "Tvoje BMI je " << BMI << endl;

  return 0;
}
